workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,44 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/root6-roofit-snakemake
+#   $ source ~/.virtualenvs/root6-roofit-snakemake/bin/activate
+#   $ pip install snakemake
+#   $ cp -a ../code ../workflow/snakemake/inputs.yaml .
+#   $ snakemake -s ../workflow/snakemake/Snakefile -p --cores 1
+
+configfile: "inputs.yaml"
```

Comment:
We can remove this for now as it may cause problems for the integration with reana. In the instructions on top we can add `--configfile ../workflow/snakemake/inputs.yaml` as I've done [here](https://github.com/reanahub/reana-demo-helloworld/pull/53/files).

Reply:
For some reason this isn't working for me, I'll look more into it

Reply:
For me, it works as it is by removing this line.